be/src/common/config.h

```diff
@@ -1522,4 +1522,6 @@ CONF_mBool(avro_ignore_union_type_tag, "false");
 // default batch size for simdjson lib
 CONF_mInt32(json_parse_many_batch_size, "1000000");
 CONF_mBool(enable_dynamic_batch_size_for_json_parse_many, "true");
+
+CONF_Bool(enable_multiple_memtable_for_lake_pindex, "false");
```

Comment:
I don't think we need to make it configurable.

Reply:
And remove `_immutable_memtable`

Reply:
> I don't think we need to make it configurable.

done